calicoctl/calico_ctl/status.py

```diff
@@ -163,6 +164,8 @@ def pprint_bird_protocols(version):
     """
     # Based on the IP version, run the appropriate BIRD command, and select
```

Comment:
I'd keep the above comment below this line, and then add a comment that says "We need to run as root so we can access the BIRD control socket"